scripts/copy-vendor.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const vendor = "src/vendor";
+
+const bootstrap_js = vendor + "/bootstrap/js";
+const bootstrap_css = vendor + "/bootstrap/css";
+
+const fontawesome_css = vendor + "/fontawesome/css";
+const fontawesome_webfonts = vendor + "/fontawesome/webfonts";
+
+const bootstrap_dist = "node_modules/bootstrap/dist";
+const fontawesome_dist = "node_modules/@fortawesome/fontawesome-free";
+
+function mkDirCatch(new_path){
+    fs.mkdir(new_path, {recursive: true}, (err) => {
+        if (err){
+            if (err.code === 'EEXIST') {
+                console.log("Path Exists " + new_path);
+                return;
+            } else {
+                console.warn("Path Create Failed " + err.code + " https://nodejs.org/api/errors.html#errors_error_code_1");
+            }
+        } else {
+            console.log("Path Created " + new_path);
+        }
+    });
+}
+
+function copyDep(src, in_dest) {
+    
+    const dest = (in_dest === undefined) ? vendor : in_dest;
+    const dep = path.basename(src);
+
+    fs.writeFile(src, dest, (err) => {
```

Comment:
You should use `fs.copyFile` to copy files. Note that destination path should contain file name:
```js
fs.copyFile(src, `${dest}/${dep}`, (err) => {
    ...
}
```

`fs.writeFile` is currently overwriting files in `node_modules` directory.

Reply:
Yes. Sorry for the delay. I should have kept it local until I knew 100%.

Reply:
```javascript
 fs.copyFile(src, dest + path.sep + dep, (err) => {
        ...
    });
```